docs/actions-reusable.md

```diff
@@ -0,0 +1,63 @@
+Like [scripted actions](actions-scripts.md), reusable actions are logical units of analytic code.
+However, whereas the source code for a scripted action must be copied from study to study, the source code for a reusable action is shared by different studies.
+
+## Running reusable actions
+
+You can browse existing reusable actions at <https://actions.opensafely.org>.
+Although each is different, they have a common API.
```

Comment:
It's never spelt out what this API is.

Reply:
Good point 🙂 I've spelt out what we mean by "a reusable API".

Reply:
@iaindillingham: If this documentation page is the canonical reference for reusable actions, you might also want to link back on <https://actions.opensafely.org> to this page once it exists. You might have already thought of this, but just in case not :slightly_smiling_face:

Reply:
Yup, that's on the list! 🙂